types/git-init/package.json

```diff
@@ -0,0 +1,21 @@
+{

```

Comment:
please remove this file, unless the intent was to add external dependencies.
In doubt, please re-run generate command as explained, to scaffold content